tools/download_posters.js

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
```

Comment:
note to myself -- this JS is not needed for frontend, it is to download all the posters, so if anything could come with `tools/create_thumb.sh`

Reply:
Yes, this could just be a simple shell script... with jq and for loop.